.github/workflows/release.yaml

```diff
@@ -27,3 +27,7 @@ jobs:
           token: "${{ secrets.WEAVEWORKSBOT_TOKEN }}"
       - name: Open PRs to release branch and default branch
         run: make prepare-release
+      - name: Publish docs to Netlify
+        run: make publish-docs
+        env:
+          NETLIFY_BUILD_HOOK_URL: ${{ secrets.NETLIFY_BUILD_HOOK_URL }}
```

Comment:
is this the right time to trigger it? It might mean the docs are changed hours before the release is out? or if circleci fails it will never come out

Reply:
> It might mean the docs are changed hours before the release is out?

This is the current behaviour as well. One of the last steps in `make prepare-release` was to force-push to `docs`, which meant that the Netlify deployment was triggered before the release was complete. This is preserving that behaviour without requiring to force-push, although we might want to consider publishing docs after the release is complete.

Reply:
ahh yeah okay. This seems like a good first fix, and then we can improve later to have it happen closer to the final release?

Reply:
Yup, I will open an issue.